<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraDefinitionDialog</class>
 <widget class="QDialog" name="CameraDefinitionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera data</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditWidth"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelCameraName">
        <property name="text">
         <string>Camera name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelCamera">
        <property name="text">
         <string>Camera:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLineEdit" name="lineEditHeight"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelWidth">
        <property name="text">
         <string>Sensor width (mm):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="labelHeight">
        <property name="text">
         <string>Sensor height (mm):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="lbWidthStimated">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLabel" name="lbHeightStimated">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QLabel" name="labelScale">
        <property name="text">
         <string>Scale factor to 35 (Crop factor) (mm):</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLineEdit" name="lineEditScale"/>
      </item>
      <item row="1" column="1" colspan="4">
       <widget class="QLineEdit" name="lineEditCameraName"/>
      </item>
      <item row="0" column="1" colspan="4">
       <widget class="QComboBox" name="comboBoxCamera"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="labelLens">
        <property name="text">
         <string>Lens:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelLensName">
        <property name="text">
         <string>Lens name:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelLocal">
        <property name="text">
         <string>Focal length (Physical) (mm):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditFocal"/>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="lbFocalStimated">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditLensName"/>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxLens"/>
      </item>
     </layout>
     <zorder>comboBoxLens</zorder>
     <zorder>lineEditFocal</zorder>
     <zorder>lineEditLensName</zorder>
     <zorder>labelLens</zorder>
     <zorder>labelLocal</zorder>
     <zorder>labelLensName</zorder>
     <zorder>lbFocalStimated</zorder>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft">
    <widget class="QFrame" name="warningFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Warning: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbWarningImg">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&lt;font color=&quot;#808000&quot;&gt;Estimated values.&lt;/font&gt; Processing results could be invalid. Please check the sensor parameters.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="aspectRatioErrorFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Error: &lt;font color=&quot;red&quot;&gt;Invalid size ratio.&lt;/font&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxCamera</tabstop>
  <tabstop>lineEditCameraName</tabstop>
  <tabstop>lineEditWidth</tabstop>
  <tabstop>lineEditHeight</tabstop>
  <tabstop>lineEditScale</tabstop>
  <tabstop>comboBoxLens</tabstop>
  <tabstop>lineEditLensName</tabstop>
  <tabstop>lineEditFocal</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CameraDefinitionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CameraDefinitionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
